keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -347,6 +349,52 @@ class SqlDeliveryConfigRepository(
       }
   }
 
+  override fun getConstraintStateForNotification(notificationId: String): ConstraintState? {
```

Comment:
I started with this name because I planned to add a `notification_id` column to the table, when I realized I could just use the `uid`. Will rename.

Reply:
Minor nit, since `notificationId == ENVIRONMENT_ARTIFACT_CONSTRAINT.UID`, might want to name this more generically, such as `getConstraintStateById`.

Reply:
Yeah, absolutely. Will rename the method and the argument! 👍